### DISCUSSION TITLE
Promote different sets of settings as modes

### DISCUSSION THREAD

**Discussion by NohWayJose** [gopro/labs#709] [Feature Requests]
See #708.
As a diver, and presumably any other sport where I can access the buttons but NOT the touch screen, it would be great to be able to switch between named setting groups within a mode. 

I suggest that the QRcontrol app allows saving sets of settings for any mode. These _sets_ could appear as modes and so be easily selectable with the mode change button. 

Using the underwater scenario to illustrate why this would be useful.
1. It's often very difficult to get the GoPro to react to the QR code (waterproof paper, if you're wondering), because light levels, plankton, etc
2. Subject needs change and so it's neccesary to change video from say a low res wide happy snap  style to 4k 60fps narrow for a bit or for photos, a wide happy snap jpg style to a RAW or different metering positions, etc

I imagine this solution could result in a lot of mode changing if someone _went to town_ on new 'modes' but it would make possible what is currently impossible, once committed to a dive. Personally, I'd probably only add those modes I mentioned above (so 2 or 3 video modes and 2 or 3 photo modes and remove time-lapse, as per #708)

**Comment by dnewman-gpsw** [comment]
AS HERO12 now supports named presets, this would somewhat address the issue. Each scuba shooting mode would have its own named preset.  The simultaneous mode+shutter press, brings up to touchless menu to select the presets. 
![image](https://github.com/gopro/labs/assets/10856084/1f883e99-0091-486c-a8c6-58fa40377449)
